widgetry/src/widgets/spinner.rs

```diff
@@ -35,7 +36,13 @@ impl Spinner {
             TEXT_WIDTH + up.get_dims().width,
             up.get_dims().height + down.get_dims().height,
         );
-
+        if current < low {
+            current = low;
+            println!("Spinner current value is out of bounds!");
```

Comment:
We're importing the log crate, so `warn!` for these. @michaelkirk and I are slowly trying to clean up `println`s

Reply:
Fair enough, wasn't sure which you wanted to use and saw `println` elsewhere.
Could maybe put something in the dev docs code conventions, to make other contributors aware?

Reply:
Will do today or tomorrow!

Reply:
Thank you!